Addition of Epiphany Assessment

Nannaquin opened the pull request.

Nannaquin committed:
Final iteration of assessment

stewartweiss reviewed (changes requested):
The evaluation is complete, but the formatting needs to be fixed. The numbering no longer works - it is all "1". See this:
![Screenshot at 2020-02-24 21-12-52](https://user-images.githubusercontent.com/39007707/75208780-a3787b80-574a-11ea-81a1-f306b3dde0ec.png)

Fix the numbering. It is probably because the answers are not indented by three spaces.

Nannaquin committed:
Fixed numbering

Manually adjusted the numbering from a stream of 1's to proper order (per section).

Nannaquin commented:
I fixed the numbering manually - the formatting seemed normal once I opened the file again.

stewartweiss reviewed (approved):
We can accept it in its present state, even though there are still minor formatting issues, with some answers not being indented. It is worth understanding why and how it should be fixed.  In a numbered list, you do not need to manually sequence. You can write:
   `1. blah blah`

   `1. blah blah`

   `1. blah blah`

and it will format in sequence. But if you interrupt this as you do like this:


1. What is the primary programming language in the project?
<br>
C and Javascript

1. What is the development environment? For example, is it Gnu C++ on Linux?
   Are there instructions for how to download, build, and install?

Downloaded, and compiled using their JHBuild program. Makes use of GNU Toolchain programs and   python runtime. 
Additionally there are instructions for building from the source in the README.md file. Utilizes the Meson build system.

the sequencing restarts at 1 and the text in between is not indented. The correct solution is to indent the lines that are answers by 3 or at most 4 spaces.
